backend/hostgw/network.go

```diff
@@ -87,15 +87,24 @@ func (n *network) handleSubnetEvents(batch []subnet.Event) {
 				continue
 			}
 
-			route := netlink.Route{
-				Dst:       evt.Lease.Subnet.ToIPNet(),
-				Gw:        evt.Lease.Attrs.PublicIP.ToIP(),
-				LinkIndex: n.linkIndex,
+			route := routes.Route{
+				Destination: evt.Lease.Subnet.ToIPNet(),
+				Gateway:     evt.Lease.Attrs.PublicIP.ToIP(),
+				LinkIndex:   n.extIface.Iface.Index,
 			}
-			if err := netlink.RouteAdd(&route); err != nil {
+
+			if err := routes.AddRoute(&route); err != nil {
+				errno, ok := err.(syscall.Errno)
+				// The Windows errno for "The object already exists" is 0x1392
+				if ok && errno == syscall.EEXIST || errno == 0x1392 {
```

Comment:
Yeah, I was weighing that. I was trying to avoid a copy/paste of this entire file, but the small differences between Windows and Linux had to leak in if I don't do that.